components/tag-list/demo/tag-list.html

```diff
@@ -86,6 +86,16 @@ <h2>Tag List with Interactive</h2>
 				</div>
 			</d2l-demo-snippet>
 
+			<h2>Tag List with custom tooltip text</h2>
+			<d2l-demo-snippet fullscreen>
+				<d2l-tag-list description="A bunch of example tags">
```

Comment:
Tried testing this component with NVDA and it didn't seem very accessible. Was only reading out the first one and then `list item 2 of 7, list item 3 of 7 etc`. This is with or without my changes. Is this expected?

Reply:
We do have an open defect about that: [DE49511](https://rally1.rallydev.com/#/15545167705ud/search?detail=%2Fdefect%2Ff1adaca6-2151-4f51-b6c8-9dbceffa3705&keywords=tag%20list&searchScopeType=project). I do think that this would play into the fix for that - I assume we'd want to always read both the text and the description to the user

Reply:
Yeah we'd want to read both for sure. From looking at the defect it looks like the behaviour noted in the defect matches what we are seeing.

Reply:
Yeah I'm not sure where along the way this broke (it definitely _used_ to work) but I'm surprised we haven't tackled it yet, seems pretty bad. I'll poke about that